src/lib/core/tests/FuzzTlvReader.cpp

```diff
@@ -4,10 +4,87 @@
 #include "lib/core/TLV.h"
 #include "lib/core/TLVUtilities.h"
 
+using namespace chip;
+using namespace chip::TLV;
+
 using chip::TLV::TLVReader;
 
-static CHIP_ERROR FuzzIterator(const TLVReader & aReader, size_t aDepth, void * aContext)
+static CHIP_ERROR FuzzIterator(TLVReader & aReader, size_t aDepth, void * aContext)
 {
+    {
+        aReader.GetLength();
+    }
+    {
+        aReader.GetTag();
+    }
+    {
+        aReader.GetType();
+    }
+    {
+        size_t size;
+        aReader.CountRemainingInContainer(&size);
+    }
+    {
+        bool v;
+        aReader.Get(v);
```

Comment:
Likly outside of scope, however I really wish getters would be `const`. A reader with non-const getters seems silly :/

Reply:
Those getters can pretty radically change the state of the reader, though....